tests/ert/ui_tests/gui/test_docs_screenshots.py

```diff
@@ -0,0 +1,464 @@
+                "has been added to the test-images artifact."
+                if IS_RUNNING_IN_GITHUB_ACTIONS
+                else "has been overwritten with the new version."
+            }
+            If the new and old image looks the same and both are correct, the test might
+            simply need to lower the similarity threshold.
+
+            If the updated image reflects an intended gui change it should be kept.
+            If the updated image seems incorrect the underlying code generating
+            the gui might need corrections or this test needs to be fixed.
+            """
+        )
+
+
+def set_data_type_selection_index(data_type_widget, index):
+    data_type_widget.data_type_keys_widget.setCurrentIndex(
+        data_type_widget.filter_model.index(index, 0)
+    )
+
+
+def clean_up_diplayed_runpath(gui: QWidget):
+    experiment_panel = gui.findChild(ExperimentPanel)
+    assert isinstance(experiment_panel, ExperimentPanel)
+    single_test_run_panel = experiment_panel.findChild(SingleTestRunPanel)
+    assert isinstance(single_test_run_panel, SingleTestRunPanel)
+    runpath_label = single_test_run_panel.findChild(CopyableLabel)
+    current_directory = os.getcwd()
+    label_text = runpath_label.label.text()
+    runpath_label.label.setText(label_text.replace(current_directory, "&lt;cwd&gt;"))
+
+
+def open_gui_with_docs_example(
+    tmp_path, source_root, example_folder, config_file, random_seed=None
+):
+    def ignore_pngs(src, files):
+        return [f for f in files if f.endswith(".png")]
+
+    shutil.copytree(
+        os.path.join(source_root, example_folder),
+        tmp_path,
+        ignore=ignore_pngs,
+        dirs_exist_ok=True,
+    )
+
+    if random_seed is not None:
+        with open(config_file, encoding="utf-8") as f:
+            original_content = f.read()
+
+        combined_content = (
+            f"RANDOM_SEED {random_seed}\n" + original_content
+        )  # Add fixed random seed to make the plots reproducible
+
+        with open(config_file, "w", encoding="utf-8") as f:
+            f.write(combined_content)
+
+    gui_generator = open_gui_with_config(tmp_path / config_file)
+    return next(gui_generator)
+
+
+def test_that_poly_new_minimal_screenshots_are_up_to_date(
+    tmp_path,
+    monkeypatch,
+    qtbot,
+    source_root,
+):
+    monkeypatch.chdir(tmp_path)
+
+    example_folder = "docs/ert/getting_started/configuration/poly_new/minimal"
+    gui = open_gui_with_docs_example(tmp_path, source_root, example_folder, "poly.ert")
+
+    if not gui.available_fonts:
+        pytest.skip(SKIP_MESSAGE)
+
+    gui_evaluator = GuiEvaluator(source_root, example_folder, gui, qtbot)
+    gui_evaluator.compare_img_with_gui("ert.png", ERT_PNG_THRESHOLD)
+
+    run_experiment(qtbot, EnsembleExperiment, gui)
+
+    gui_evaluator.compare_img_with_gui("simulations.png", SIMULATIONS_PNG_THRESHOLD)
+
+    assert not gui_evaluator.gui_change_detected(), gui_evaluator.change_report()
```

Comment:
General comment, there seems to be a slight change of style. In some of the tests there are widely used empty lines. I don't argue over the amount itself, more that it seems inconsistent between how the code is formatted. Just a thought.